library/Backends/llvm/test/Test/Codegen.hs

```diff
@@ -33,20 +34,62 @@ runIntModule mod =
             Just f -> runIntFun f
             Nothing -> P.error "no main function found"
 
+llvmIntType :: Word32 -> PassTypes.TypeLLVM
+llvmIntType size = PassTypes.PrimTy $ Primitive.PrimTy $ AST.IntegerType {typeBits = size}
+
+llvmIntVal :: Integer -> PassTypes.TermLLVM
+llvmIntVal = PassTypes.Prim . Primitive.LitInt
+
+llvmAnnotatedTerm :: PassTypes.TypeLLVM -> PassTypes.TermLLVM -> PassTypes.Annotated PassTypes.TermLLVM
+llvmAnnotatedTerm ty term = PassTypes.Ann {PassTypes.usage = Usage.SAny, PassTypes.annTy = ty, PassTypes.term = term}
```

Comment:
Check my `codegen` module, I'm fairly certain I have a function for their `AST.IntegerType` though I may be wrong

Reply:
There are interfaces for `IntegerType`s with specific sizes -- `i4`, `i8`, and so on -- but I wanted an interface which produced a `TypeLLVM` and which was parameterized by `typeBits` since I'm constructing `TermLLVM`s manually in test code.